### ISSUE TITLE
License Change Consent (from MIT to LGPL-3.0 or later)

### ISSUE THREAD

**Issue by EliteAsian123** [YARC-Official/YARG#601]
### YARG will soon be changing it's license from MIT to LGPL-3.0 (or any version later)

LGPL is a more restrictive license compared to MIT, however it's fully compatible. See details [here](https://www.tldrlegal.com/license/gnu-lesser-general-public-license-v3-lgpl-3).

Before proceeding with this license update, we want to ensure that we have your consent. If you are in agreement with the proposed change, **__please reply to this message with your approval__**. If you have any concerns or objections, we value your feedback and would be happy to discuss further.

Your understanding and cooperation are highly appreciated as we work towards the continued growth of YARG. Thank you for your ongoing support!

### Why?

Some of the reasons are: LGPL requires that any modified version of the software (in this case, YARG), must also have their source code published as LGPL. LGPL also has provisions related to patents, which prevents (to an extent) the use of patents to restrict the use of the software (please read the LGPL license for more details regarding these things if interested), among other things.

### Will this affect any plans with the project, or any libraries?

Nope. LGPL will still allow us to do things such as close-sourcing certain parts of the project (such as anti-cheat), and will still allow use to use libraries with proprietary licenses (such as BASS).

### Who?

95% (which in this case is 33/34 listed) of everyone who has made code and art contributions to YARG must give consent. Grammatical changes in various markdown files have been excluded. Those people are as follows:
* ✅ @TheNathannator
* ✅ @RileyTheFox
* ✅ @TheFatBastid 
* ✅ @raphaelgoulart 
* ✅ @rjkiv
* ✅ @muskit
* ✅ @jnackmclain
* ✅ @JayDiddyThaGOAT
* ✅ @kaduwaengertner
* 🟪 @berthrage (no response to changes; 95% threshold achieved, license change will proceed)
* ✅ @nevespt
* ✅ @EscapeNumber001
* ✅ @santorfo
* ✅ @SamuelPalma
* ✅ @sonicfind
* ✅ @LocalH
* ✅ @Pantotone
* 🟦 @thoudankeykang (all contributions moved to OpenSource, permission not needed)
* ✅ @NyxTheShield
* ✅ @LlysiX
* ✅ @grishhung
* ✅ @NarrikSynthfox
* ✅ @Roystermeat
* 🟦 @jtanan44 (all contributions overridden as of now; this is a non-retroactive change, permission not needed)
* ✅ @eckerj
* ✅ @PikminGuts92
* ✅ @zigguy
* ✅ @yoitsOJ
* ✅ @vcinventerman
* ✅ @PixelatedPope
* ✅ @InvoxiPlayGames
* ✅ @ieee802dot11ac
* ✅ @NicolasPL64
* ✅ @exoticri

Any objections from these people will be worked out before this change is made. Thanks for your contributions and understanding.

### TL;DR

Please give approval below of whether you not you consent of YARG changing licenses from MIT to LGPL-3.0 (or any version later) if you have been mentioned above.

**Comment by Pantotone** [comment]
LGTM! 👍

**Comment by EscapeNumber001** [comment]
I consent to this change.

**Comment by santorfo** [comment]
All good from me 👌

**Comment by TheNathannator** [comment]
I fully endorse this change 👍

**Comment by jnackmclain** [comment]
You have my approval

**Comment by NarrikSynthfox** [comment]
Sounds good to me

**Comment by vcinventerman** [comment]
All good with me 👍

**Comment by raphaelgoulart** [comment]
I consent to this change.

**Comment by yoitsOJ** [comment]
Fully support this change 👍

**Comment by PixelatedPope** [comment]
Yup. No issues here

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: EliteAsian ***@***.***>
Sent: Tuesday, November 21, 2023 2:06:00 PM
To: YARC-Official/YARG ***@***.***>
Cc: PixelatedPope ***@***.***>; Mention ***@***.***>
Subject: [YARC-Official/YARG] License Change Consent (from MIT to LGPL-3.0 or later) (Issue #601)


YARG will soon be changing it's license from MIT to LGPL-3.0 (or any version later)

LGPL is a more restrictive license compared to MIT, however it's fully compatible. See details here<https://www.tldrlegal.com/license/gnu-lesser-general-public-license-v3-lgpl-3>.

Before proceeding with this license update, we want to ensure that we have your consent. If you are in agreement with the proposed change, please reply to this message with your approval. If you have any concerns or objections, we value your feedback and would be happy to discuss further.

Your understanding and cooperation are highly appreciated as we work towards the continued growth of YARG. Thank you for your ongoing support!

Why?

Some of the reasons are: LGPL requires that any modified version of the software (in this case, YARG), must also have their source code published as LGPL. LGPL also has provisions related to patents, which prevents (to an extent) the use of patents to restrict the use of the software (please read the LGPL license for more details regarding these things if interested), among other things.

Will this affect any plans with the project, or any libraries?

Nope. LGPL will still allow us to do things such as close-sourcing certain parts of the project (such as anti-cheat), and will still allow use to use libraries with proprietary licenses (such as BASS).

Who?

95% (which in this case is everybody) of everyone who has made code and art contributions to YARG must give consent. Those people are as follows:

  *   🟨 @TheNathannator<https://github.com/TheNathannator>
  *   🟨 @RileyTheFox<https://github.com/RileyTheFox>
  *   🟨 @TheFatBastid<https://github.com/TheFatBastid>
  *   🟨 @raphaelgoulart<https://github.com/raphaelgoulart>
  *   🟨 @rjkiv<https://github.com/rjkiv>
  *   🟨 @muskit<https://github.com/muskit>
  *   🟨 @jnackmclain<https://github.com/jnackmclain>
  *   🟨 @JayDiddyThaGOAT<https://github.com/JayDiddyThaGOAT>
  *   🟨 @kaduwaengertner<https://github.com/kaduwaengertner>
  *   🟨 @berthrage<https://github.com/berthrage>
  *   🟨 @nevespt<https://github.com/nevespt>
  *   🟨 @EscapeNumber001<https://github.com/EscapeNumber001>
  *   🟨 @santorfo<https://github.com/santorfo>
  *   🟨 @SamuelPalma<https://github.com/SamuelPalma>
  *   🟨 @sonicfind<https://github.com/sonicfind>
  *   🟨 @LocalH<https://github.com/LocalH>
  *   🟨 @Pantotone<https://github.com/Pantotone>
  *   🟨 @thoudankeykang<https://github.com/thoudankeykang>
  *   🟨 @NyxTheShield<https://github.com/NyxTheShield>
  *   🟨 @LlysiX<https://github.com/LlysiX>
  *   🟨 @grishhung<https://github.com/grishhung>
  *   🟨 @NarrikSynthfox<https://github.com/NarrikSynthfox>
  *   🟨 @Roystermeat<https://github.com/Roystermeat>
  *   🟨 @jtanan44<https://github.com/jtanan44>
  *   🟨 @eckerj<https://github.com/eckerj>
  *   🟨 @PikminGuts92<https://github.com/PikminGuts92>
  *   🟨 @zigguy<https://github.com/zigguy>
  *   🟨 @yoitsOJ<https://github.com/yoitsOJ>
  *   🟨 @vcinventerman<https://github.com/vcinventerman>
  *   🟨 @PixelatedPope<https://github.com/PixelatedPope>
  *   🟨 @InvoxiPlayGames<https://github.com/InvoxiPlayGames>
  *   🟨 @ieee802dot11ac<https://github.com/ieee802dot11ac>
  *   🟨 @NicolasPL64<https://github.com/NicolasPL64>
  *   🟨 @exoticri<https://github.com/exoticri>

Any objections from these people will be worked out before this change is made. Thanks for your contributions and understanding.

TL;DR

Please give approval below of whether you not you consent of YARG changing licenses from MIT to LGPL-3.0 (or any version later) if you have been mentioned above.

**Comment by RileyTheFox** [comment]
I consent to this change!

**Comment by SamuelPalma** [comment]
I completely agree with the changes 👌

**Comment by exoticri** [comment]
I consent to this change

**Comment by PikminGuts92** [comment]
I consent to the license change 👍

**Comment by eckerj** [comment]
Good for me :-)

**Comment by TheFatBastid** [comment]
I consent to this change. Rock on!

**Comment by muskit** [comment]
I consent to the license change.

**Comment by kaduwaengertner** [comment]
I consent to the license change.

**Comment by nevespt** [comment]
I consent to the license change.

**Comment by sonicfind** [comment]
*Cracks knuckles*
![image](https://github.com/YARC-Official/YARG/assets/52436993/aedf12aa-c5a5-4cdf-9922-7cf5a28b6775)

**Comment by LlysiX** [comment]
sounds good to me bossman

**Comment by NicolasPL64** [comment]
Sure thing! I consent

**Comment by LocalH** [comment]
Fine with me

**Comment by Roystermeat** [comment]
I consent

**Comment by zigguy** [comment]
I consent

**Comment by ieee802dot11ac** [comment]
👍

**Comment by NyxTheShield** [comment]
I consent!

**Comment by rjkiv** [comment]
👍🏻

**Comment by InvoxiPlayGames** [comment]
awesome change, i consent

**Comment by JayDiddyThaGOAT** [comment]
I consent to this change!

**Comment by thoudankeykang** [comment]
although permission isn't required from me, i still do consent
